src/FrenetOptimalTrajectory/FrenetOptimalTrajectory.cpp

```diff
@@ -63,6 +94,154 @@ FrenetPath* FrenetOptimalTrajectory::getBestPath() {
     return best_frenet_path;
 }
 
+
+/* 
+ * Threaded version of calc_frenet_paths
+ * We parallelize on the outer loop, in terms of di
+ * Here it iterate over di indexes, from start index up to but not include the end index.
+ * It then computes the actual di value for path planning. 
+ */
+void FrenetOptimalTrajectory::calc_frenet_paths_threaded(int start_di_index, int end_di_index) {
+    double t, ti, tv;
+    double lateral_deviation, lateral_velocity, lateral_acceleration, lateral_jerk;
+    double longitudinal_acceleration, longitudinal_jerk;
+    FrenetPath* fp, *tfp;
+    int num_paths = 0;
+    int num_viable_paths = 0;
+    // double valid_path_time = 0;
```

Comment:
Remove commented line.

Reply:
There are a few more comments following it that also use `valid_path_time`, and they times the valid path; so I decided to keep it.